appyters/L1000FWD_Consensus_Drugs/README.md

```diff
@@ -2,6 +2,20 @@
 
 This Appyter takes up and down gene sets and returns the consensus mimicker and reverser drugs associated to them. It sends the up and down gene sets to [L1000FWD](https://amp.pharm.mssm.edu/l1000fwd/) to get the top 50 mimicker and reverser signatures. It then counts the number of times a drug appear in the top 50 signatures of a query gene set. The appyter only keeps drugs that appear in a certain percentage of query gene sets defined by **drug percentage**. The p-value for each drug is then computed using a pre-computed empirical distribution using [CREEDS](https://amp.pharm.mssm.edu/creeds/#downloads) gene sets as background signatures. The drugs are then visualized using [clustergrammer](https://amp.pharm.mssm.edu/clustergrammer/). A publication ready static heatmap is also provided for the users.
 
+## File Format
+The appyter takes up and down gene sets gmt files with the following format:
+```
+signature 1\t\tGENE 1\tGENE 2\tGENE 3
+signature 2\t\tGENE 3\tGENE 4\tGENE 5
+signature 3\t\tGENE 2\tGENE 5\tGENE 6
+signature 4\t\tGENE 1\tGENE 2\tGENE 6
+```
+
+Signature names should be _the same_ on both up and down gmt files. Below are the sample up and down gene sets:
+
+* [Up Gene Set](https://appyterbucket.s3.amazonaws.com/sample_data/up_down_signatures/up_diseases)
```

Comment:
Is this somewhere on our cloud or do we need to get this in the appyter s3?

Reply:
It's in our cloud. I have an s3 bucket on Alex's account I created a while ago for my example files. Do we want to move it to the appyter s3 you made?

Reply:
It's fine given that it is in our cloud. It might be useful to move it into appyters when you get a chance so that it's in one place. If we want to move this system eventually (i.e. to an internal system) it will be easier if we don't need to search for stray files.